cluster/cluster-authentication.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cluster-authentication
```

Comment:
the name can be better. something explicit is better. (as in siamak's comment)

shall we make it `kubeconfig-writter` 
or `kubeconfig-creator`

cc @siamaksade